docs/quickstarts/sso-quickstart-customize.md

```diff
@@ -0,0 +1,642 @@
+
+7. In the **User.Read** row of the permissions table, choose the ellipsis and then select **Remove permission** from the menu that appears.
+
+8. Select the **Yes, remove** button in response to the prompt that's displayed.
+
+9. Select the **Add a permission** button.
+
+10. On the panel that opens choose **Microsoft Graph** and then choose **Delegated permissions**.
+
+11. On the **Request API permissions** panel:
+
+    a. Under **Files**, select **Files.Read.All**.
+
+    b. Select the **Add permissions** button at the bottom of the panel to save these permissions changes.
+
+12. Select the **Grant admin consent for [tenant name]** button.
+
+13. Select the **Yes** button in response to the prompt that's displayed.
+
+## Try it out
+
+If your add-in is an Excel, Word, or PowerPoint add-in, complete the steps in the following section to try it out. If your add-in is an Outlook add-in, complete the steps in the [Outlook](#outlook) section instead.
+
+### Excel, Word, and PowerPoint
+
+Complete the following steps to try out an Excel, Word, or PowerPoint add-in.
+
+1. In the root folder of the project, run the following command to build the project, start the local web server, and sideload your add-in in the previously selected Office client application.
+
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. In the Office client application that opens when you run the previous command (i.e., Excel, Word or PowerPoint), make sure that you're signed in with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while [configuring SSO](sso-quickstart.md#configure-sso) for the app. Doing so establishes the appropriate conditions for SSO to succeed. 
+
+3. In the Office client application, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. The following image shows this button in Excel.
+
+    ![Excel add-in button](../images/excel-quickstart-addin-3b.png)
+
+4. At the bottom of the task pane, choose the **Read my OneDrive for Business** button to initiate the SSO process. 
+
+5. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft Account or Office 365 ("Work or School") account. Choose the **Accept** button in the dialog window to continue.
```

Comment:
Would step 5 ever happen in the context of this article? They've already been told to use the admin account in the Azure portal and they've been told to sign on to Office with "a user that's a member of the same Office 365 organization" (so they are not using a Microsoft Account).

Reply:
If they follow instructions, step 5 wouldn't happen. But as you pointed out previously (when suggesting that info about signing in to Azure be formatted as bold text, for emphasis) -- some folks will inevitably *not* follow the instructions, in which case they'd see this dialog. I've included this information here so people won't feel completely derailed if/when this happens, and also for the purpose of illustrating that the add-in has the 'fallback auth flow' capability in situations where SSO requirements aren't met.

Reply:
OK